docs/source/format/Columnar.rst

```diff
@@ -404,6 +404,9 @@ All integers (length, buffer index, and offset) are signed.
 
 This layout is adapted from TU Munich's `UmbraDB`_.
 
+Note that this layout uses one additional buffer to store the variadic buffer
+lengths in the :ref:`Arrow C data interface <c-data-interface-binary-view-arrays>`.
```

Comment:
```suggestion
lengths in the :ref:`Arrow C Data Interface <c-data-interface-binary-view-arrays>`.
```

Reply:
I first had it that way, but then changed to lower case because that's the casing we are using on that page itself .. ;)
(https://arrow.apache.org/docs/dev/format/CDataInterface.html)

Reply:
Ah, fair enough :-)